pkg/preparer/orchestrate.go

```diff
@@ -94,6 +98,7 @@ func (p *Preparer) WatchForPodManifestsForNode(quitAndAck chan struct{}) {
 				go p.handlePods(podChanMap[podId], quitChanMap[podId])
 			}
 			podChanMap[podId] <- result.Manifest
+			errs <- nil
```

Comment:
So you are saying I should test the case where `StatusPort` is not configured. OK, will do. Testing it in universe.